index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
+    <title>Car Lot</title>
+
+    <!-- Bootstrap -->
+    <link href="/lib/node_modules/bootstrap/dist/css/bootstrap.min.css" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="styles/quiz.css">
+  </head>
+  <body>
+
+    <div class="container">
+    	<h1>Check out our cars!</h1>
+
+    			<nav class="navbar" id="topNavbar">
+    				<!-- <a class="navbar-brand" href="#"> -->
+    				<ul class="nav navbar-nav">
+
+    					<li class="nav-item"><img src="images/lotlogo.jpg"</a></li>
+    					<!-- <li class="nav-item active">
+    						<a class="nav-link" href="#">Home <span class="sr-only">(current)</span></a>
+    					</li>
+    					<li class="nav-item">
+    						<a class="nav-link" href="#">Home</a>
+    					</li>
+    					<li class="nav-item">
+    						<a class="nav-link" href="#">Home</a>
+    					</li>
+    					<li class="nav-item">
+    						<a class="nav-link" href="#">Home</a>
+    					</li> -->
+    				</ul>
+    				<div class="form-inline navbar-form pull-right">
+    					<input class="form-control" type="text" placeholder="Update the description" id="userInput">
+    					<button class="btn btn-success-outline" type="submit" id="clearBTN">Clear</button>
+    				</div>
+    			</nav>
+
+    			<div class="row" id="output"><!-- use these in my javascript DOM handler
+    				  <div class="col-md-4">Column testing</div>
+
+    				  <div class="col-md-4">Column testing</div>
+
+    				  <div class="col-md-4">Column testing</div> -->
+
+
+
+    			</div>
+
+    </div>
+
+
+
+    <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
```

Comment:
If you are using a package manager there is no reason not to use `npm install jquery` to install dependencies directly into your project. Remember to `--save` if you want them written into your package.json file.